tools/minikube/instant-demo.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+set -u
+
+: "${USE_SSL:=false}"
+: "${backend_proto:="http"}"
+: "${S3GW_CHARTS:="$(dirname "$0")/../../charts/charts/s3gw"}"
+
+if [[ ! -r "${S3GW_CHARTS}/Chart.yaml" ]]; then
+    echo "S3GW_CHARTS set to something weird ($S3GW_CHARTS)"
+    echo "Please set to the s3gw charts dir (e.g s3gw-charts/charts/s3gw)"
+    exit 1
+fi
+
+set -x
+
+minikube start --driver=kvm2 --extra-disks=0
+
+helm repo add jetstack https://charts.jetstack.io
+helm repo add traefik https://traefik.github.io/charts
+
+helm repo update
+
+if [[ $USE_SSL == "true" ]]; then
+    backend_proto="https"
+    helm install cert-manager \
+	 --namespace cert-manager jetstack/cert-manager \
+	 --create-namespace \
+	 --set installCRDs=true \
+	 --set extraArgs[0]=--enable-certificate-owner-ref=true
+fi
+helm install traefik-ingress traefik/traefik
+
+traefik_cluster_ip="$(kubectl \
+			      get service \
+			      traefik-ingress -ojsonpath='{.spec.clusterIP}')"
+
+cat > /tmp/s3gw-helm-values.yaml <<EOF
```

Comment:
Please add a cleanup function so this script doesn't litter artifacts all over the place.

Reply:
:+1: